### ISSUE TITLE
Userlist crashes Browser (long loading)

### ISSUE THREAD

**Issue by ExaconAT** [nextcloud/server#18219]
### Steps to reproduce
1. Login as admin
2. Open User List (430+ Users)
3. Browser hangs and i see the following output in the Console

### Expected behaviour
User list should load fast.

### Actual behaviour
User list loads slow and any Browser ond any System hangs

### Server configuration

**Operating system:** Ubuntu 16.04 Latest Patch Level

**Web server:** Apache Latest

**Database:** Mysql Latest

**PHP version:** 7.3.12

**Nextcloud version:** 17.0.1

**Updated from an older Nextcloud/ownCloud or fresh install:** 16.0.4

**Where did you install Nextcloud from:** Update from CLI

**Signing status:**
<details>
<summary>Signing status</summary>

```
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- core
	- EXTRA_FILE
		- landingpage_css/style.css
		- landingpage_css/style_act_reg.css
		- featherlight/bower.json
		- featherlight/.gitignore
		- featherlight/.bowerrc
		- featherlight/assets/fonts/glyphicons-halflings-regular.eot
		- featherlight/assets/fonts/glyphicons-halflings-regular.ttf
		- featherlight/assets/fonts/glyphicons-halflings-regular.woff
		- featherlight/assets/fonts/glyphicons-halflings-regular.svg
		- featherlight/assets/stylesheets/bootstrap.min.css
		- featherlight/assets/images/red_and_yellow.jpg
		- featherlight/assets/images/two_birds.jpg
		- featherlight/assets/images/droplets.thumb.jpg
		- featherlight/assets/images/blue.thumb.jpg
		- featherlight/assets/images/red_and_yellow.thumb.jpg
		- featherlight/assets/images/feed.jpg
		- featherlight/assets/images/feed.thumb.jpg
		- featherlight/assets/images/black.jpg
		- featherlight/assets/images/blue.jpg
		- featherlight/assets/images/black.thumb.jpg
		- featherlight/assets/images/droplets.jpg
		- featherlight/assets/images/two_birds.thumb.jpg
		- featherlight/assets/javascripts/jquery-1.7.0.min.js
		- featherlight/.travis.yml
		- featherlight/featherlight.jquery.json
		- featherlight/test/featherlight_gallery.html
		- featherlight/test/fixtures/photo.jpeg
		- featherlight/test/fixtures/photo_large.jpg
		- featherlight/test/featherlight_test.js
		- featherlight/test/featherlight.html
		- featherlight/test/.jshintrc
		- featherlight/test/featherlight_gallery_test.js
		- featherlight/test/helpers/patiently.js
		- featherlight/LICENSE
		- featherlight/.github/ISSUE_TEMPLATE.md
		- featherlight/.github/CONTRIBUTING.md
		- featherlight/.jshintrc
		- featherlight/gallery.html
		- featherlight/release/featherlight.min.js
		- featherlight/release/featherlight.min.css
		- featherlight/release/featherlight.gallery.min.css
		- featherlight/release/featherlight.gallery.min.js
		- featherlight/package.json
		- featherlight/CHANGELOG.md
		- featherlight/Gruntfile.js
		- featherlight/README.md
		- featherlight/src/featherlight.gallery.js
		- featherlight/src/featherlight.gallery.css
		- featherlight/src/featherlight.css
		- featherlight/src/.jshintrc
		- featherlight/src/gallery.html
		- featherlight/src/featherlight.js
		- featherlight/src/index.html
		- featherlight/index.html
		- landingpage_img/cloud_transfer.png
		- landingpage_img/portallogo_alt.png
		- landingpage_img/iphone_cloud.png
		- landingpage_img/icon_app_store.png
		- landingpage_img/cloud_good.png
		- landingpage_img/icon_play_store.png
		- landingpage_img/arrow_yellow.png
		- landingpage_img/Schwinge.png
		- landingpage_img/cloud_security.png
		- landingpage_img/portallogo.png
		- landingpage_img/blue_bar_cloud.png
		- landingpage_img/gruen_haken.png
		- landingpage_img/zuhause_img.jpg
		- landingpage_img/zuhause_img.png
		- landingpage_img/arrow_grey.png
		- landingpage_img/cloud_header_image.png
		- landingpage_img/arrow_white.png
		- landingpage_img/logo_300.png

Raw output
==========
Array
(
    [core] => Array
        (
            [EXTRA_FILE] => Array
                (
                    [landingpage_css/style.css] => Array
                        (
                            [expected] => 
                            [current] => c0127a6f23cd59fb7a5e679fc36cf849fbbf1160a0559def261c5dbc1dbe1a43cc41b34df025eaa6ac917851140463b22ff94d479bdec91cfd494b8d10f11dcc
                        )

                    [landingpage_css/style_act_reg.css] => Array
                        (
                            [expected] => 
                            [current] => e06a8dae5e6f055c751c99c2132f490139ed4b3fe5c2d0f42936e7a2e7149eb8705f345fa912834b2c84cceeecd894f12c08065efb9fd5bd78eb1b65089263bf
                        )

                    [featherlight/bower.json] => Array
                        (
                            [expected] => 
                            [current] => 9f36610f6aa8ad0037804edcffaffac77acecdb7789b5cff93792f7766557847dc2ee40b259d100c3e11deab2fbd3391f331ffda98d6f69b7228c752bec24b1e
                        )

                    [featherlight/.gitignore] => Array
                        (
                            [expected] => 
                            [current] => 4d29c23020d36de89e1c6206e9cc5fee2b51e3e3f4fbb88e01f5ed289b2a1893cdb72172292dc59443f88979fcad0a806b1c677bfdde014b61e766be5c426d49
                        )

                    [featherlight/.bowerrc] => Array
                        (
                            [expected] => 
                            [current] => 479198a9802dd8358214c76925e219baaf69734bd300cbf31d81682a863a2c9e7f414d2752adc31bf3723ef9843797cc498cf94862b897889fdb9085ee462251
                        )

                    [featherlight/assets/fonts/glyphicons-halflings-regular.eot] => Array
                        (
                            [expected] => 
                            [current] => 830f534f97a8eb849fa7e67e533d89d150ca79233c0a31a9a54f27b4757a86c345f94533c41fc239899e642502fde7800170e6889d1544de69bd6809ade8387b
                        )

                    [featherlight/assets/fonts/glyphicons-halflings-regular.ttf] => Array
                        (
                            [expected] => 
                            [current] => f47c250403fbb7f2dfe1cb7068ea3a1496fd26d9df6bfc3a701d3909c5a2b69f43bab042df0dcb66e64c308945745b76b0e9ad2a750c69cc0c7271c6d6051a96
                        )

                    [featherlight/assets/fonts/glyphicons-halflings-regular.woff] => Array
                        (
                            [expected] => 
                            [current] => 88beda715a3de8ea031048feb71fd3ffff9c695fd9874b06c8b1d31c88c7a8a653c4a67efab3b6f0b1f4dedc3c18a6436a4455984c0abce10a210baca0cb689d
                        )

                    [featherlight/assets/fonts/glyphicons-halflings-regular.svg] => Array
                        (
                            [expected] => 
                            [current] => 94b9b2263796425a74b86f521dc255071e04f587664929972aa67270b1b2572a791360d575a4da8146779f1a8ca2aaf41f24130881ee7e973d2862c5cb0b5f73
                        )

                    [featherlight/assets/stylesheets/bootstrap.min.css] => Array
                        (
                            [expected] => 
                            [current] => 2a7f594ff849d49936246e8744037eff8cb79aa06d3fb0ef3885882896143f07c04b9b5b0669cc3b11129603e0ab829004a95e35becaae3c71d86e1dac0dad2f
                        )

                    [featherlight/assets/images/red_and_yellow.jpg] => Array
                        (
                            [expected] => 
                            [current] => d0cb25b63ca3b0ff87aa83ea2142ea81d42e28e7d965e27a797cde5b9b22eeaeb929332ac66216eee3e42d17a83e390eee1d610570ae948d4953bc86574aa1d4
                        )

                    [featherlight/assets/images/two_birds.jpg] => Array
                        (
                            [expected] => 
                            [current] => 62f1d54a793b6c89ff136ec3c60d2c3edbb5b4c1ed6a8d53c84337c4e1a01308a8d0f6d70961aeeeeb583dc2228f3bbb80e8e49fe7d81c96e645b274a0307ae1
                        )

                    [featherlight/assets/images/droplets.thumb.jpg] => Array
                        (
                            [expected] => 
                            [current] => aa436dbc9d2f1979fdb3bd29c2044593532f9291bcb56ebedffa399b86296a2fd5000e8b8c32d18fe74d65b906cba1c4692a54417016d31f31ef6e08d713f627
                        )

                    [featherlight/assets/images/blue.thumb.jpg] => Array
                        (
                            [expected] => 
                            [current] => 5866f0544e4f91679e5c52f6074810985205a113cca0d7ff276068f78538b8049965c3d4983ea55204dd3ac3f30246689584f22a834c36a04b299e0bc6cd5f93
                        )

                    [featherlight/assets/images/red_and_yellow.thumb.jpg] => Array
                        (
                            [expected] => 
                            [current] => 4f33c445996dcfe629330a8f2497bc9f0e0fc1df2569c8f0846bc82a851fa77a68bd05e1d36efe1cb8dab98a3651147aa476df899fd2865991a7175374413d68
                        )

                    [featherlight/assets/images/feed.jpg] => Array
                        (
                            [expected] => 
                            [current] => a6ea64b3872986d1a8a64aaf3a59cd6aab601b0dacdaad396a2999b1c3c7372a2b977740c6db35c2edc6f28b32aadc494c9aaf60d2ec321bc6573a84a3d5b325
                        )

                    [featherlight/assets/images/feed.thumb.jpg] => Array
                        (
                            [expected] => 
                            [current] => 4d621fd68b25ee76a2c843e7fcc4145e0992ecc73cb2d9052a8085c220f82d21e42af76430d931459f470e67d69c49736d71a2f3c99f3745ff4e1f999fc5e245
                        )

                    [featherlight/assets/images/black.jpg] => Array
                        (
                            [expected] => 
                            [current] => 9669297f85c163b108a6c928b068ba214a4469df5aa4ee466ace07bb39207baf2ce3fab32f6241cf261a5fd80dda0bef7fd58c7c5f78e741d5527896c493725c
                        )

                    [featherlight/assets/images/blue.jpg] => Array
                        (
                            [expected] => 
                            [current] => 7767e06aa1a23e6760738807b5867fbab3cbf8293fba071b727f3e106ab2802837ac162486ca4b0c951ee8b5ae30aed7e26b88a9af012abb35e7be40181aa002
                        )

                    [featherlight/assets/images/black.thumb.jpg] => Array
                        (
                            [expected] => 
                            [current] => d9d49ec112d8b4d0b3cc26cc69dc7e7318f1de81171a94ba2c613f811ba54bbcbabe377c1020128a9e40933cc2d73eb653db59453db5cbc395edab3092733771
                        )

                    [featherlight/assets/images/droplets.jpg] => Array
                        (
                            [expected] => 
                            [current] => 891ba87fa5197915e03cc06c1d7396e8868bbfc2769810ada1cd6ead69cc3f038c85f062eccfc8cd7ba489991fe440742c9b1a37cf1f3cef1bf181cbc38e1579
                        )

                    [featherlight/assets/images/two_birds.thumb.jpg] => Array
                        (
                            [expected] => 
                            [current] => 33371bae69448fa2555c335915c21177d37650716e6d924a932e442092ae3af49e6d8ab0c8940753566651b39060883849d8064a87cc76b364279efb4f6d1c9d
                        )

                    [featherlight/assets/javascripts/jquery-1.7.0.min.js] => Array
                        (
                            [expected] => 
                            [current] => 3eac74655a1c1c24b34cb2b8a66d0685c707b4e4fa7e6673fada9f60757f96523ad2007db2bf63922aaba9213014922bce97191073c98ecab819c0bbdf550d5e
                        )

                    [featherlight/.travis.yml] => Array
                        (
                            [expected] => 
                            [current] => f5e80011039cc714dd2ff09c4f1ceceb9121ae31f9e1e90066f771d39479e7263e40fa3bc02ff204a09ff02f6e629479ea4b979c5f1017be79d9b7a73acf97c5
                        )

                    [featherlight/featherlight.jquery.json] => Array
                        (
                            [expected] => 
                            [current] => a2829b4cf3e9fa87a34cbf4fe610ce76e1e7afab706afa9263f69b092f8186d085c5d95b0c91bbb4a0d80d8eec90e37b5a6cdf756081d255bf7d397a7252a841
                        )

                    [featherlight/test/featherlight_gallery.html] => Array
                        (
                            [expected] => 
                            [current] => 0e12a3496b3a8c514a78dcc695f207920d4804786765da98cb565ea509ab2be97b59b9afb9e9bef3d133d01b389595cd9fa004e4acf1647a9a711cfee5041c8c
                        )

                    [featherlight/test/fixtures/photo.jpeg] => Array
                        (
                            [expected] => 
                            [current] => 3f8bdd19bfd2654d9370f5494f9174da39c59c3a7e45c52c98f2a30b57a83a1743faf296d78da9f6d571956732ee251613cfc241354409c3d1ab9dcd02a09c67
                        )

                    [featherlight/test/fixtures/photo_large.jpg] => Array
                        (
                            [expected] => 
                            [current] => 86cf946266fde28892a9fcad719e3dae9cc13cd0b0cca7f71ec6c4dad21cc5a11ebae0ae0e4b8e1dab4dcde7929819074f611b293c216a8055edfea3d7bfecb7
                        )

                    [featherlight/test/featherlight_test.js] => Array
                        (
                            [expected] => 
                            [current] => 3326757e42ad5b9a2555a0ffc574479496d65266c9740848abdb2d52ed2010df35dfb5696691a205d8fab8a4e363395eabe5cd5f4ad90973406409439f6dc641
                        )

                    [featherlight/test/featherlight.html] => Array
                        (
                            [expected] => 
                            [current] => d3c162e868b74a81660fbe2b73e899a1f396bdbef44d1d56862f688a56113d14f2f035454be5d5182a19a8e754ac545cc16577ea8e4a9cdc268a8c9813f40dce
                        )

                    [featherlight/test/.jshintrc] => Array
                        (
                            [expected] => 
                            [current] => edde02e1df37568ac7f96132f77d3e4587a563769a29957d034ec0440cd0f481435a12eaabd140aed557bc58971a9252568670dc58a0d7dcf243308d9ef29d88
                        )

                    [featherlight/test/featherlight_gallery_test.js] => Array
                        (
                            [expected] => 
                            [current] => efca613cee76ff9592341e792e94b66820cf052bb644da914420cbdcc7ce8613810d5a31e7f58e35adba69e3fe55c233259d2d05fe4d4ddf5fb3aa18288bb091
                        )

                    [featherlight/test/helpers/patiently.js] => Array
                        (
                            [expected] => 
                            [current] => 2ce5fc42b8aa9af7db702d3a31b949c6b9d6af2918a41608e704f82d33039eeef69bd6877ad6640a10cf8117bfead13fdfeb68fe2fad8a21e3201dee9a5c439c
                        )

                    [featherlight/LICENSE] => Array
                        (
                            [expected] => 
                            [current] => fc8abacc45d4fffe510dddfbb50fd769a2bb1e3fdcb95857fc8f87406b9244d3b30e4dff2737c889923c5c99c38cf46d06f9a52f368c718edbd2a1e83e43a699
                        )

                    [featherlight/.github/ISSUE_TEMPLATE.md] => Array
                        (
                            [expected] => 
                            [current] => 29ca4c205a4e6a873090bc2ac00fdfabd3b7333dc1896fc4b0394a68035b67e3ff7e121474f1c638d27b33b820858a15dab098800935c9bc31ae3c81f970a11d
                        )

                    [featherlight/.github/CONTRIBUTING.md] => Array
                        (
                            [expected] => 
                            [current] => 9dc74fc4f445be234c7a5888b4d95e792c0a30a987e3fd4f1cea997102e5388153104eb31a6023d2f6fbde6ecec57ac6c64c57d82fcfb2d7f3ef7b0b16d1fef6
                        )

                    [featherlight/.jshintrc] => Array
                        (
                            [expected] => 
                            [current] => 965c5dc6c98b7716a2f42d6fe7cb0625e808dbf9b3c8c03d792e930b3ee290294a1d7a3b8671cdfe952822ed6e1a23fd8d39b9fe65362300a5158f5e2217be11
                        )

                    [featherlight/gallery.html] => Array
                        (
                            [expected] => 
                            [current] => 6b2339f7aebf67944a3ce7f1f449bad285f501d34b8251f3728521a168d496cb3506b057e4b6d4d7a73428cbbb023ea6425f0703d7e2b733a402d15138128181
                        )

                    [featherlight/release/featherlight.min.js] => Array
                        (
                            [expected] => 
                            [current] => 72ae5dc5c5b918adfbeef00db59ffdce465791eec1d1d8df0de38cd6449fdafbaa4d39783d4fa9509a7e9c5b3022fb0d2ccbea4246279d8d4fa80a83c9af72b5
                        )

                    [featherlight/release/featherlight.min.css] => Array
                        (
                            [expected] => 
                            [current] => e240bf1ef56a15618969bbcc5a884b45f2a664cb3c843d99a5d7c52497329efee6e10f80f5307f5c1a3dc37f06fe00ea4058f7cd907888ee14553bd0a694dbdc
                        )

                    [featherlight/release/featherlight.gallery.min.css] => Array
                        (
                            [expected] => 
                            [current] => ffe093a9be5810bc76010945761f55f9fc8ef039c42d65c60cbb9e569efa96a76cffc7519041b6d8c95999042603ee00bc7ece3bbcd1e20c6ccc0aa576ee2566
                        )

                    [featherlight/release/featherlight.gallery.min.js] => Array
                        (
                            [expected] => 
                            [current] => b81f050863161fab8e72e00b981667e716d2bc5caa10a2f06e9a5e53e2749d16c44fa6d810190f163cadbfdaec43ec1a33ed7156e369e6eab532c78c571a5273
                        )

                    [featherlight/package.json] => Array
                        (
                            [expected] => 
                            [current] => ea527682daa8610cedfe1243722326fa09c0038ccff8db5c3db08592d785f031087ada1ca6e7b19aeaadcd1b986161e68a72319ed4fe675ff4e3cf6db7fd239a
                        )

                    [featherlight/CHANGELOG.md] => Array
                        (
                            [expected] => 
                            [current] => 8c35a7b68edc14213e4ef43282cd2a549871caa5997110a992710d3a80d8313f7371eb9574cbcc76b8104ccf6609be44efb7833685729be541cd8bcafe726c0d
                        )

                    [featherlight/Gruntfile.js] => Array
                        (
                            [expected] => 
                            [current] => fd5bd29f78d6662c86353dc69334d02cc86ed5c05aa9e510bc99c4ffda0ed27b6faafbfe5a18f1299c3c8cad9c679257036a8a93b8e4e6a962764a20512afc41
                        )

                    [featherlight/README.md] => Array
                        (
                            [expected] => 
                            [current] => c512750243a516ff2aa314bd959dec372d750b8ee1b0c7612e14d5bc441ab10907a17b9d826ce5eeaf0bc79624fe06400f1ab32572ffa8b89698ed5190d30b9d
                        )

                    [featherlight/src/featherlight.gallery.js] => Array
                        (
                            [expected] => 
                            [current] => 786f4bc08327f8f6b6ea4fe960b39e33cb2681c5d7ee28efd0790bcc576e881cbce4d29e96904cdcbe88b868273c6380b627dfbc20498eabb61ed75c96d76ef9
                        )

                    [featherlight/src/featherlight.gallery.css] => Array
                        (
                            [expected] => 
                            [current] => c749bc0f7b20cee3606eb0e6aa1d697c9697a92ddb1a170f1a5624a83628ffb7a7c6fd6e24866837f1ff6b93bbd353a2a8dc8cc941c9674a3352b090a0912e72
                        )

                    [featherlight/src/featherlight.css] => Array
                        (
                            [expected] => 
                            [current] => b8c0a78495e12a29f99991971ad3873a89b746cbec9206c8c89c032733e510342ea51e3f48c741a36827dd74c09775b7ae963135c413ef79181b5eefa0050784
                        )

                    [featherlight/src/.jshintrc] => Array
                        (
                            [expected] => 
                            [current] => b9204f486f21c97f4bd5b6e4cf5367e4a8878011d2fc82af891526419f0f6b24a45b17be2262943fd8cd3a4622994371c9687d922d0937969a47f20ac913c15d
                        )

                    [featherlight/src/gallery.html] => Array
                        (
                            [expected] => 
                            [current] => 115982a431e4e3f289be33e3d3fe8c8ec123536d8e335297db0c53b9721d59f408a268f5e97946b41424d75bb67e49ddcbf98750483d88135caaa91ba764807c
                        )

                    [featherlight/src/featherlight.js] => Array
                        (
                            [expected] => 
                            [current] => 966f88451888d04c2e9ba1efedacb182e849f47030321750878231bc36e17f41a8191b767b9900511a81ca2913e38256804de27102b4e557ed5da217b1c0ca73
                        )

                    [featherlight/src/index.html] => Array
                        (
                            [expected] => 
                            [current] => 24ad8d30100c2282cc81c0c21c7fc9fefd77af64d1f215ad4b3084ff06fe60c3f8252753147395cdff595afc75424563e0267c0d45d866378c5d447c32346590
                        )

                    [featherlight/index.html] => Array
                        (
                            [expected] => 
                            [current] => 519c97baaa5101072e3b754b8ce97eb02dfe5a528c8bb0926cf7bd604447bd1cfa70a8c58491be3e1a5e7d11d4b2f22cb7d10dfc824e00439a9930ea18a93ade
                        )

                    [landingpage_img/cloud_transfer.png] => Array
                        (
                            [expected] => 
                            [current] => bb200f548b8135326387a2a289ff87a656d0c0b0f481578a6cac8ad2521a4d1c520019d5d7ba681ceb8df74b5f1dca3a9338c6be6badef3063347729391b1497
                        )

                    [landingpage_img/portallogo_alt.png] => Array
                        (
                            [expected] => 
                            [current] => ac04685ddfed8336f61e41f2067355d9a4f5e931a51cb4e0d976e20b98acac58e44ac61dbbf8d3da6715cd372952a348d567b155a656c7fb512171d23f873136
                        )

                    [landingpage_img/iphone_cloud.png] => Array
                        (
                            [expected] => 
                            [current] => a4da6a671a85c1a46ef36318bc85de5ad1a38abd3583f63573a805829baafcc1d5c9972377c247c94b073efbbcab6c9fc05be0bede546555b78dc9f11226ea56
                        )

                    [landingpage_img/icon_app_store.png] => Array
                        (
                            [expected] => 
                            [current] => f8fd6402a2f4ff1ae929847e08979c2568b80d8b2107117b26558e29d7a756c038ea0bfdbbab08c34bb4456fff18f58ab60e58f5699da109db31b67b3b9afd5e
                        )

                    [landingpage_img/cloud_good.png] => Array
                        (
                            [expected] => 
                            [current] => cf364f89ef0390e0709897629c4a0c85bf1f832e5bfd5068bbafa50c4ef0915f4d18d22b8a7f33a51463cc04d2d2dfa8fa76023266ba5cdd88e6e63392ec0131
                        )

                    [landingpage_img/icon_play_store.png] => Array
                        (
                            [expected] => 
                            [current] => 75b1f577f2db9d036e4800add0994995797fcef0c02c8ab3949ee285d07fc706af19259531b80c4bea081e167fc3c107c5c8e82d4e47443f27411a3dec962fdb
                        )

                    [landingpage_img/arrow_yellow.png] => Array
                        (
                            [expected] => 
                            [current] => cfb4f53f2953149d60e336c153adb1dc3ef55f45ceed2c59480c2821d80f22031c5ae1beff70d9db190160c01e10761d5a9551bcdc166f9c8f007a2212f4e62c
                        )

                    [landingpage_img/Schwinge.png] => Array
                        (
                            [expected] => 
                            [current] => 6e47bc39bacb243b88b6683138602a7f2b84493a521d01a0b6f30938a579cad38b9133b25fe10ba50e167a91d60acfdd048146e0e228ab1cecda8a248680eefc
                        )

                    [landingpage_img/cloud_security.png] => Array
                        (
                            [expected] => 
                            [current] => 8b36d700dfce21e76c02c18baaa097021bacfc8ecad2fe34410b004d08a044e229b74d07ac968557b0a3d2069928c8bb08a57a2715585435f3cbd000d1b3f8ce
                        )

                    [landingpage_img/portallogo.png] => Array
                        (
                            [expected] => 
                            [current] => a34717ca15c7fad310421b5a85463343e23cec38452492eee4fd67c95f0f91ebcd6e520ce8801c8c61af820bf3e26947359ea8cc97543c89e0947b40f37bf443
                        )

                    [landingpage_img/blue_bar_cloud.png] => Array
                        (
                            [expected] => 
                            [current] => 55213e996973b8a8efd2b4194b4b07dc969fdf93f0e1f0273ed5161f64738db36816c050cf43c24d83e8d5d5411b99439bd26110e56609f2455dba625d374e23
                        )

                    [landingpage_img/gruen_haken.png] => Array
                        (
                            [expected] => 
                            [current] => 1d4c29d055587eb0d5830d8dc50e05895a254f4f3dec98e69c395af4110679eff556149141d430915921788e60a54865136a27196ad6a298a58cb7d52ad30323
                        )

                    [landingpage_img/zuhause_img.jpg] => Array
                        (
                            [expected] => 
                            [current] => 8ecca983c7e8edf085ef9d481805dd8f6498c8e149e1ed9577d1a4862401f5db5f1fae72688cddbd1ddb5aca22d0c5f9ac411ced5ed03fa14122a68d51d7956a
                        )

                    [landingpage_img/zuhause_img.png] => Array
                        (
                            [expected] => 
                            [current] => f19f8f035d0f627e8aa536e2ffbeab071ecdeeada01f92e84c663c507885be2b14ccb8bea8e2b874a4a38f164322b11a7d569689a529eaa509292381b156ef50
                        )

                    [landingpage_img/arrow_grey.png] => Array
                        (
                            [expected] => 
                            [current] => 61178c106312c6f06f693b62cd9a3671f03289704f4d134180314d548e1142d9747f4dc45b5a8a8e6a8a2a4fb16f07212281d8e23fec619d9ff2a469a4c0a73b
                        )

                    [landingpage_img/cloud_header_image.png] => Array
                        (
                            [expected] => 
                            [current] => 34b515815abff233c20ab7f10eb7ad4a88775be9744dd68c66c90ae1178be8576c61832e3b0be878064b301320b94ef784b3769c81e3782457b9fda0c041fd29
                        )

                    [landingpage_img/arrow_white.png] => Array
                        (
                            [expected] => 
                            [current] => e2caab1e605404e02565223dd6a8e71076ae4d6bb86f63942c19e60b637141f7cee0db857729f12ad3279c9884e63d42926b67d45cc7a0e00768e5d5e69f83fd
                        )

                    [landingpage_img/logo_300.png] => Array
                        (
                            [expected] => 
                            [current] => 338e81a621c23569a86b46161ee8248cbbe9543e139134dbcfdde7f85eb21934b0cd1dc3bf085674e35353bdbcc4a17065ea7be32729e84a63bc9ad86588598a
                        )

                )

        )

)

```
</details>

**List of activated apps:**
<details>
<summary>Enabled:
  - accessibility: 1.3.0
  - activity: 2.10.1
  - admin_audit: 1.7.0
  - bruteforcesettings: 1.4.0
  - cloud_federation_api: 1.0.0
  - comments: 1.7.0
  - dav: 1.13.0
  - deck: 0.7.0
  - external: 3.4.1
  - federatedfilesharing: 1.7.0
  - federation: 1.7.0
  - files: 1.12.0
  - files_pdfviewer: 1.6.0
  - files_rightclick: 0.15.1
  - files_sharing: 1.9.0
  - files_trashbin: 1.7.0
  - files_versions: 1.10.0
  - files_videoplayer: 1.6.0
  - firstrunwizard: 2.6.0
  - gallery: 18.4.0
  - groupfolders: 5.0.4
  - impersonate: 1.4.0
  - logreader: 2.2.0
  - lookup_server_connector: 1.5.0
  - notifications: 2.5.0
  - oauth2: 1.5.0
  - onlyoffice: 3.0.2
  - password_policy: 1.7.0
  - privacy: 1.1.0
  - provisioning_api: 1.7.0
  - serverinfo: 1.7.0
  - sharebymail: 1.7.0
  - support: 1.0.1
  - systemtags: 1.7.0
  - text: 1.1.1
  - twofactor_backupcodes: 1.6.0
  - updatenotification: 1.7.0
  - workflowengine: 1.7.0
</summary>
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***",
            "***REMOVED SENSITIVE VALUE***",
            "***REMOVED SENSITIVE VALUE***",
            "***REMOVED SENSITIVE VALUE***",
            "127.0.0.1"
        ],
        "0": [
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "17.0.1.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "appstore.experimental.enabled": false,
        "mail_smtpmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "loglevel": 1,
        "maintenance": false,
        "ldapIgnoreNamingRules": false,
        "theme": "***REMOVED SENSITIVE VALUE***",
        "updater.release.channel": "stable",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "overwritecondaddr": "***REMOVED SENSITIVE VALUE***",
        "mail_template_class": "***REMOVED SENSITIVE VALUE***",
        "skeletondirectory": "",
        "mysql.utf8mb4": true,
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "mail_sendmailmode": "smtp"
    }
}
```
</details>

**Are you using external storage, if yes which one:** Nope

**Are you using encryption:** Nope

**Are you using an external user-backend, if yes which one:** Nope

```

### Client configuration
**Browser:** Firefox, Edge... Latest

**Operating system:** Windows 10 latest patch level

### Logs
#### Web server error log
<details>
<summary>Web Server ACCESS/ERROR LOG</summary>

```
**No errors, or special ACCESS information.**
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
**No errors. / Info**
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
**For every user:

The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. 4 globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. 2 globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. 5 globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. globals.js:26:15
The escapeHTML library is deprecated! It will be removed in nextcloud 19. 2 globals.js:26:15**
```
</details>

**Comment by kesselb** [comment]
Sounds like https://github.com/nextcloud/server/issues/17903 that is fixed by https://github.com/nextcloud/server/pull/18142 and will be shipped with 17.0.2.

**Comment by ExaconAT** [comment]
> 
> 
> Sounds like #17903 that is fixed by #18142 and will be shipped with 17.0.2.

Working with fix now, but slow loading. ~ 5 seconds for 25 Users.

**Comment by dertorsten** [comment]
Here slow with ~100 Users and fix applied. Firefox complains about script slowing down browser, Chrome loads but slow.

**Comment by skjnldsv** [comment]
I will need a screenshot of your network requests.
Before you do anything, still on the development tools, there is a tab called `network`. Click on it and then the xhr filter. Do your action and screenshot the network log like this:
![capture d ecran_2018-09-27_21-49-46](https://user-images.githubusercontent.com/14975046/46170857-43548000-c29f-11e8-9403-db72713dd9c1.png)

**Comment by kesselb** [comment]
Fixed with 17.0.2
